core/src/com/fallenflame/game/GameEngine.java

```diff
@@ -790,12 +781,6 @@ public boolean scrolled (int amount) {
         if(!isScreenActive()){
             return true;
         }
-        if(amount == 1 && ALLOW_DEBUG){
```

Comment:
Why is this causing the bug/ This should never be true unless we have ALLOW_DEBUG, as the light radius was a useful debugger